src/render/tutorials/caching-http-requests/2_0.html.md

```diff
@@ -0,0 +1,337 @@
+---
+title: Caching HTTP requests
+author: oskarjerzyk
+keywords: tutorial
+date: 2019-10-21
+layout: tutorial
+knotxVersions:
+    - 2.0.0
+---
+# Overview
+In this tutorial we will add ability to cache http request to project based on [Knot.x Starter Kit](https://github.com/Knotx/knotx-starter-kit) 
+template.
+
+We will use existing `ActionFactory` from [knotx-fragments](https://github.com/Knotx/knotx-fragments) to achieve our goal
+which will be implementing custom endpoint with cache ability.
+
+## Prerequisites
+- JDK 8
```

Comment:
We've forgotten about Gradle, in other tutorials too. Can you please add it here and check other tutorials (a separate PR please).

Reply:
is Gradle really needed if we use `gradlew`?

Reply:
I think it's not needed, gradlew will download appropriate gradle version and we don't care about it